### ISSUE TITLE
Invoke-Parallel should work with PS strict mode

### ISSUE THREAD

**Issue by vors** [RamblingCookieMonster/Invoke-Parallel#5]
```powershell
Import-Module .\Invoke-Parallel.ps1 -Force
Set-StrictMode -Version latest

@(1,2,3) | Invoke-Parallel { $_ }
```
![image](https://cloud.githubusercontent.com/assets/816680/5865297/3bd615f4-a23d-11e4-942f-c7aa9b9cf24f.png)

**Comment by RamblingCookieMonster** [comment]
With this fixed, PowerShell 2.0 strict mode will fail with $runspace.powershell.HadErrors.  Didn't realize this property didn't exist in v2.  Will refactor the HadErrors side, revisit this.

**Comment by RamblingCookieMonster** [comment]
In testing the haderrors piece for this, noticed errors are sent to standard output - would it be worth writing these to the error stream (e.g. loop through streams.error and Write-Error -ErrorRecord $_)?

**Comment by vors** [comment]
Definitely!

**Comment by vors** [comment]
Fixed in #11
